irods/test/connection_test.py

```diff
@@ -169,6 +169,11 @@ def test_assigning_session_connection_timeout_to_large_values__issue_623(self):
         # safe for whenever a Pool sub-object is initialized (or re-initialized).
         self.assertEqual(sess._cached_connection_timeout, MAXIMUM_CONNECTION_TIMEOUT)
         
+        # Make (and check) a change of the connection_timeout value so that second of the surrounding
+        # equality assertions does not accidentally succeed due to the value remaining untouched.
+        sess.connection_timeout = 1
+        self.assertNotEqual(sess._cached_connection_timeout, MAXIMUM_CONNECTION_TIMEOUT)
```

Comment:
Would it be more appropriate to check what the value *is* equal to (would it be 1?) rather than what it is *not* equal to?

Reply:
But testing equality will be just as good.